crates/sb_ai/lib.rs

```diff
@@ -11,45 +11,52 @@ use tokenizers::Tokenizer;
 
 deno_core::extension!(
     sb_ai,
-    ops = [op_sb_ai_run_model],
+    ops = [op_sb_ai_run_model, op_sb_ai_init_model],
     esm_entry_point = "ext:sb_ai/ai.js",
     esm = ["ai.js",]
 );
 
-fn run_gte(state: &mut OpState, prompt: String) -> Result<Vec<f32>, Error> {
+fn init_gte(state: &mut OpState) -> Result<(), Error> {
```

Comment:
It would be interesting if we can make this op `async` and have the whole logic of initialization (if it's heavy) on a different thread. (I'm thinking especially because models can be large)

Reply:
@andreespirela Thanks for the suggestion! Refactored the init to run on a tokio task (ie. green thread)